package.json

```diff
@@ -143,6 +143,7 @@
     "eslint-plugin-prettier": "^3.1.1",
     "eslint-plugin-react": "^7.16.0",
     "eslint-plugin-unicorn": "^12.1.0",
+    "eslint-plugin-custom-rules": "file:eslint",
```

Comment:
How does this work?

Reply:
This is basically telling where to look for the custom ESLint plugin which is in "eslint" folder that I added. So when running yarn, the plugin will be copied into node_modules folder. More detailed explanation can be found in the first article that I referenced.

Reply:
That's very cool. I'm only really worried about how this works in CI and/or on Windows machines.

Reply:
Ah okay. I suppose I could publish this as a npm package. What do you think?

Reply:
Let's see how it works for us!